lib/ancestry/class_methods.rb

```diff
@@ -40,14 +40,14 @@ def arrange options = {}
     # @param nodes [Array[Node]] nodes to be arranged
     # @returns Hash{Node => {Node => {}, Node => {}}}
     # If a node's parent is not included, the node will be included as if it is a top level node
-    def arrange_nodes(nodes)
+    def arrange_nodes(nodes, add_orphans: true)
```

Comment:
It might make more sense to call is `rootify_orphans` or perhaps in the reverse `discard_orphans`?